data-prepper-api/src/main/java/org/opensearch/dataprepper/model/configuration/SinkModel.java

```diff
@@ -132,24 +131,18 @@ void validateConfiguration() {
             }
         }
 
-
         /**
-         * Pre-processes a list of Keys and returns a sorted list
-         * The keys must start with `/` and not end with `/`
-         *
-         * @param keys a list of raw keys
-         * @return a sorted processed keys
+         * Validates both include and exclude keys if they contain /
          */
-        private List<String> preprocessingKeys(final List<String> keys) {
-            if (keys.contains("/")) {
-                return new ArrayList<>();
-            }
-            List<String> result = keys.stream()
-                    .map(k -> k.startsWith("/") ? k : "/" + k)
-                    .map(k -> k.endsWith("/") ? k.substring(0, k.length() - 1) : k)
-                    .collect(Collectors.toList());
-            Collections.sort(result);
-            return result;
+        private void validateKeys() {
+            includeKeys.forEach(key -> {
+                if(key.contains("/"))
+                    throw new InvalidPluginConfigurationException("include_keys cannot contain /");
+            });
+            excludeKeys.forEach(key -> {
+                if(key.contains("/"))
+                    throw new InvalidPluginConfigurationException("exclude_keys cannot contain /");
```

Comment:
Does this mean we don't support nested keys? seems like it should be supported like we do for `delete_entries` processor

Reply:
We decided not to support it for now, and add it later. 
https://github.com/opensearch-project/data-prepper/issues/3163#issuecomment-1681140905

Reply:
We already do not support it.

So this change just validates that users don't try to use this. See this comment and thread for more details:

https://github.com/opensearch-project/data-prepper/issues/3163#issuecomment-1679087068